src/components/TokenHolderList/TokenHolderList.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo } from 'react';
```

Comment:
this file was only renamed, it used to be for the Membership Directory. Now it can be used to render a list of `TokenHolder`s and therefore shared between Membership Directory + Community Page